src/Apps/MyCollectionInsights/myCollectionInsightsRoutes.tsx

```diff
@@ -16,6 +17,19 @@ export const myCollectionInsightsRoutes: AppRouteConfig[] = [
   {
     path: "/my-collection/median-sale-price-at-auction/:artistID",
     getComponent: () => MyCollectionInsightsMedianSalePriceAtAuction,
+    onClientSideRender: () => {
+      MyCollectionInsightsMedianSalePriceAtAuction.preload()
+    },
+    query: graphql`
```

Comment:
I believe the query is missing the variables from the URL, it needs a `prepareVariables` field similar to [this one](https://github.com/artsy/force/blob/main/src/Apps/Artwork/artworkRoutes.tsx#L19)

Reply:
this is a good comment, the url params seem to be automatically injected while respecting the format they're defined as. You can see in the video below how they're being injected properly without having the `prepareVariables` method defined. `prepareVariable` is working as middleware and it wasn't needed here. We might need it later if we want read the medium at a further point

Reply:
I forgot to link this video, as you see, params are getting injected here without the middleware

https://user-images.githubusercontent.com/11945712/197558099-cfe9f175-58f4-43bc-bc40-e6e250b0e373.mov